src/grid/rtransform.py

```diff
@@ -24,14 +24,19 @@
 from abc import ABC, abstractmethod
 from numbers import Number
 
-from grid.basegrid import OneDGrid, RadialGrid
+from grid.basegrid import OneDGrid
 
 import numpy as np
 
 
 class BaseTransform(ABC):
     """Abstract class for transformation."""
 
+    @property
+    def domain(self):
```

Comment:
I think ``Domain`` should be a class property, and ``codomain`` should be the proper term here.

Reply:
The domain is not a property of the transform, because it depends on the input provided to the transform.